.github/workflows/commit_artifacts.yml

```diff
@@ -348,10 +444,91 @@ jobs:
           grep -rl "$LAST_VERSION" ./compiled-rn || echo "No files found with $LAST_VERSION"
           grep -rl "$LAST_VERSION" ./compiled-rn | xargs -r sed -i -e "s/$LAST_VERSION/$CURRENT_VERSION/g"
           grep -rl "$LAST_VERSION" ./compiled-rn || echo "Version re-applied"
-      - name: Will commit these changes
+      - name: Add files
         if: steps.check_should_commit.outputs.should_commit == 'true'
         run: |
           echo ":"
+          git add .
+      - name: Signing files
```

Comment:
Now that the version has been switched back to the new version, we also need to re-sign the files.